src/components/notifications/notifications.scss

```diff
@@ -0,0 +1,10 @@
+.Toastify__toast {
+    overflow-wrap: break-word;
+}
+
+.Toastify__toast-body {
+    min-width: 0;
+    div {
+        min-width: 0;
+    }
+}
```

Comment:
I would not recommend overriding the component's styles. There are simpler ways of achieving this. This is also a global style, so it affects all usage of `Toastify`, not just the ones in the `Notifications` component. It should be scoped to the component if you were to do something like this.